cmd/sonobuoy/app/operations/gen_plugin_test.go

```diff
@@ -0,0 +1,41 @@
+package operations
+
+import (
+	"bytes"
+	"testing"
+
+	"k8s.io/api/core/v1"
+	kuberuntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func TestLoadNonexistantPlugin(t *testing.T) {
+	err := GeneratePluginManifest(GenPluginConfig{
+		Paths:      []string{"./plugins.d"},
+		PluginName: "non-existant-plugin",
+	})
+
+	if err.Error() != "expected 1 plugin, got 0" {
+		t.Errorf("unexpected or no error %q", err)
+	}
+}
+
+func TestLoadRealPlugin(t *testing.T) {
+	b := &bytes.Buffer{}
+	err := GeneratePluginManifest(GenPluginConfig{
+		// Tests are executed with cwd set to their containing directory
+		Paths:      []string{"../../../../plugins.d"},
```

Comment:
I'd like us to not hard code the paths if we can avoid it.  Perhaps using SONOROOT var or something.